js/interface.js

```diff
@@ -1931,6 +1931,24 @@ $('input[name="exclude"]').on('tokenfield:createtoken', function(event) {
   });
 });
 
+$('body').on('change', '.tokens-list', function() {
```

Comment:
Add comments to show what this does and how it works

Reply:
When we click on the manage security rule button from API tokens dialog box that time we filtering the rules according to selectedTokenId [ex. t1] and saving in current ds rules. Now when we changing the selectedTokenId [ex.t2 or t3] then again we have to filter the rules from currentFinalRules. according to t2 and save in current rules. 

Please see the below videos:
Before Fix:

https://user-images.githubusercontent.com/102005879/196406799-9e8aeee9-513a-4c98-987b-e42caec73517.mp4

After Fix:

https://user-images.githubusercontent.com/102005879/196406845-fc337c2e-09f8-45f5-9f07-7d59bec70e16.mp4

**Note:** while making video facing network issue so UI render slowly.

Reply:
As with my other comment, we need inline comments added to this code. Refer to your colleagues if you don't know how to write good inline comments.